### PR TITLE
Add: Matrix: -rowsDo, -colsDo

### PR DESCRIPTION
**Pull request by joslloand**
Author: Michael McCrea (@mtmccrea)

### CODE REVIEW

File: classes/various/Matrix.sc [hunk 1]
```diff
@@ -302,6 +302,20 @@ Matrix[slot] : Array {
 		});
 	}
 
+	/*
+	rowsDo
+	colsDo
+
+	Michael McCrea, 2019.
```
Comment (telephon) [inline classes/various/Matrix.sc]: it's OK to add the autor to the code, but we normally don't do this a lot, relying on the commit messages in git. But whatever you prefer.

Comment (mtmccrea) [reply classes/various/Matrix.sc]: I don't have a strong preference :)

Comment (joslloand) [reply classes/various/Matrix.sc]: Ah... yes... I'd gotten carried away here...

I'll remove the comments and just refer to @mtmccrea in the commit message.

**Comment by joslloand** [comment]
@telephon, @mtmccrea authorship credit removed from code. Credited in commit message, instead. I think this now should be ready to merge?

**Pull request merged**